website/docs/components/reveal/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,15 @@
+## Conformance rating
+
+<Doc::Badge @type="success">Conformant</Doc::Badge>
```

Comment:
(issue, blocking) It should be conditionally conformant, because we offer a way to change the text when the user interacts with the button. In these cases, the reveal toggle button is not conformant.

Reply:
Thanks, I updated the rating and added an explanation. Please take a look and let me know if I need to modify the explanatory text in any way.